### PR TITLE
Added arabic translation

### PR DESCRIPTION
**Pull request by mosabalrsaheed**
Add supports for Arabic, without RTL support.

### CODE REVIEW

File: frontend/app_flowy/assets/translations/ar.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
+  "appName": "AppFlowy",
+  "defaultUsername": "أنا",
+  "welcomeText": "مرحبًا بك في @: appName",
+  "githubStarText": "Star on GitHub",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:5]: start on Github = أضف نجمة للمشروع في Github

File: frontend/app_flowy/assets/translations/ar.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  "subscribeNewsletterText": "Subscribe to Newsletter",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:6]: Subscribe to Newsletter = إشترك في النشرة الإخبارية

File: frontend/app_flowy/assets/translations/ar.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  "letsGoButtonText": "Let's Go",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:7]: Lets Go = البدء

File: frontend/app_flowy/assets/translations/ar.json [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    "rename": "Rename",
+    "delete": "حذف",
+    "duplicate": "مكرر"
+  },
+  "blankPageTitle": "صفحة فارغة",
+  "newPageText": "صفحة جديدة",
+  "trash": {
+    "text": "المهملات",
+    "restoreAll": "استعادة الكل",
+    "deleteAll": "Delete All",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:53]: Delete all = حذف الكل

File: frontend/app_flowy/assets/translations/ar.json [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    "pageHeader": {
+      "fileName": "اسم الملف",
+      "lastModified": "Last Modified",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:56]: Last Modified = آخر تعديل

File: frontend/app_flowy/assets/translations/ar.json [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+  },
+  "deletePagePrompt": {
+    "text": "هذه الصفحة في المهملات",
+    "restore": "استعادة الصفحة",
+    "deletePermanent": "الحذف نهائيًا"
+  },
+  "DialCreatePageNameHint": "اسم الصفحة",
+  "questionBubble": {
+    "whatsNew": "ما الجديد؟",
+    "help": "Help & Support",
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:68]: Help & support = الدعم و طلب المساعدة

File: frontend/app_flowy/assets/translations/ar.json [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+    "debug": {
+      "name": "معلومات التصحيح",
+      "Success": "تم نسخ معلومات تصحيح الأخطاء إلى الحافظة!",
+      "fail": "تعذر نسخ معلومات التصحيح إلى الحافظة"
+    }
+  },
+  "menuAppHeader": {
+    "addPageTooltip": "أضف صفحة داخل الصفحة بسرعة",
+    "defaultNewPageName": "بدون عنوان",
+    "renameDialog": "Rename"
```
Comment (ibraheeamm) [inline frontend/app_flowy/assets/translations/ar.json:78]: Rename = إعادة تسمية

**Review by ibraheeamm — COMMENTED** [review]
Good translations, you need to add the missing ones @mosabalrsaheed I commented on them

Thanks

**Comment by mosabalrsaheed** [comment]
Dear @ibraheeamm, Translations updated with the missing keys.

**Comment by appflowy** [comment]
> Dear @ibraheeamm, Translations updated with the missing keys.

Hi @mosabalrsaheed , I think your PR should not carry my changes. Maybe you could open a new PR and submit your changes.
image

**Comment by mosabalrsaheed** [comment]
Sure thing @appflowy, will open another PR.

**Comment by MikeWallaceDev** [comment]
@mosabalrsaheed did you open another PR?  Could you please reference here, and then close this PR. Thank you :)

**Pull request closed without merging**